src/TideOperation.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+class TideOperation {
+  public static function install_webform_config() {

```

Comment:
I've made this a static because I don't foresee a reason that this class will need to be instantiated, but happy to change it.